.github/workflows/integration_tests.yml

```diff
@@ -1,34 +1,47 @@
 name: integration-tests
 
 on:
-  pull_request:
-    types: [ edited, synchronize ]
-  pull_request_review:
-    types: submitted
+  workflow_dispatch:
+  issue_comment:
+    types: [created]
 
 jobs:
-  check_approve_count:
+  should_run_it:
     runs-on: ubuntu-latest
     outputs:
-      approve_count: ${{ steps.reviews.outputs.approved }}
+      run_integration_tests: ${{ steps.should-run-step.outputs.should_run }}
     steps:
-      - name: Count approve count
-        id: reviews
-        run: |
-          PR_NUMBER=$(echo $GITHUB_REF | awk 'BEGIN { FS = "/" } ; { print $3 }')
-          echo $PR_NUMBER
-          echo $GITHUB_REF
-          export count=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/flank/flank/pulls/${PR_NUMBER}/reviews?per_page=100 -s | grep -i -c '"state": "APPROVED"')
-          echo $count
-          echo "::set-output name=approved::${count}"
-
-      - name: Print current approve count
-        run: |
-          echo "Current Approve count ${{ steps.reviews.outputs.approved }}"
+      - uses: khan/pull-request-comment-trigger@master
+        if: ${{ github.event_name == 'issue_comment'}}
+        id: check_issue_comment
+        with:
+          trigger: '@flank-it'
+          reaction: rocket
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+# TEMPORARY DISABLED UNTIL WE FIGURE OUT THE BEST METHOD TO RUN INTEGRATION TESTS
```

Comment:
Will this comment go into master?  I can see the comment. I am just wondering if this test is for master or only for this branch?

Reply:
I think that for now, it should be on master until we finish `Improve integration tests` epic

Reply:
Please check my comment https://github.com/Flank/flank/pull/1342#issuecomment-732885063
I think we could uncomment this action

Reply:
Could you give more info about this?
From my point of view, we should not require integration tests to pass until we fully finished it